src/manifest-creation.ts

```diff
@@ -3,6 +3,7 @@ import yaml from "js-yaml";
 import path from "path";
 import updateDotenv from "update-dotenv";
 import { ProbotOctokit } from "./octokit/probot-octokit";
+import { OctokitOptions } from "@octokit/core/dist-types/types.d";
```

Comment:
do not import internal types, that should not be necessary. This path could change any moment as it's an implementation detail. I think there is a typescript helper to extract arguments from a constructor

Reply:
extracting now OctokitOptions from ProbotOctokit constructor param